### ISSUE TITLE
Update Status Mate?

### ISSUE THREAD

**Issue by VeltabexHD** [Foxxite/MultiCharacter#5]
Hey, hope your doin well wanted to ask if its possible to for you to update this plugin to the latest version, its been sometime now & I really love to have this on my Server.

Hope everything is alright with you, wanted to leave this msg to get a what's up.

Sincerely Veltabex.

**Comment by Foxxite** [comment]
I'm really busy with personal stuff. So I don't know when my plugins will be updated. Pull request to my open source plugins for updates are welcome :)

**Comment by X-Niter** [comment]
> I'm really busy with personal stuff. So I don't know when my plugins will be updated. Pull request to my open source plugins for updates are welcome :)

I forked your project and got it updated to 1.16.5 without an issue, however, I test my code all the way to clear out as many bugs as possible sense it helps the issue tracker stay clear of silly bugs that could of been caught before hand. 
Once I work out of the basic kinks, and core functionality the plugin states on spigot, I'll send over a clean PR. 
Though I just want to clarify that you're still cool with a simple version update PR? 
I have to clear up one more thing... Your plugin is premium on spigot, though in order to test to make this is functioning, I have to build the jar which means using a prem plugin at 0 cost, is that going to conflict with you or will the update PR suffice to this prem plugin cost? Let me know, I'll delete it after words in good faith if needed. :)

**Comment by Veltabex** [comment]
I got a notification from this thread did you delete him ? Cus I wanted to try an updated version if possible!
![image](https://user-images.githubusercontent.com/62775096/133299468-5a3021a2-82be-4b7a-bf60-ae150e85b24e.png)

**Comment by Foxxite** [comment]
I did not get a notification of that message myself. But yes a clean PR of a version update is fine. 
Concerning the using a premium plugin for free, anyone is free to build the source and use it personally. The spigot page is meant as a donation to me and for people that don't have that knowhow,m similar to Citizens.. I might add builds of the jar on this repo at a later date.

**Comment by Veltabex** [comment]
Thanks boss!
Also since I have you on, I went to his page I can see the files and download it as zip, but was wondering how to do you make it into a .jar to run the file.

**Comment by Foxxite** [comment]
Compile it with maven. You might need to install the Lombok addon for your idea.

**Comment by X-Niter** [comment]
Hey @Foxxite, sorry about that, I didn't know the message was gone. However, that message does stand true, I've made quite some progress on this, just a matter of time until the major bugs are ironed out. Sorry for the lack of speed y'all, I got 2 lil ones my self and all. I am working on it though!!

**Comment by X-Niter** [comment]
> Thanks boss!
> Also since I have you on, I went to his page I can see the files and download it as zip, but was wondering how to do you make it into a .jar to run the file.

You're more then welcomed to test out my latest commits, but please if at all possible spare the time of not doing that, I can tell you right now in its 1.16.5 state it will throw some nasty stack traces, I'd hate to have questions on them when they are clear as day over on this end. :)

**Comment by Veltabex** [comment]
Thanks for the heads up, I would love to try an working version one day for 1.16.5 & 1.17.

**Comment by Foxxite** [comment]
Thanks to @X-Niter the plugins has been updated on 1.16.5. An 1.17 update will be much more challenging.

**Comment by X-Niter** [comment]
> Thanks to @X-Niter the plugins has been updated on 1.16.5. An 1.17 update will be much more challenging.

Haha, you're not wrong, I looked at 1.17.... haha

**Comment by Veltabex** [comment]
Hey that sounds wonderful I would love to try it on my dead project that I'm trying to lift up, do you think I could get a complied  copy if possible ?

**Comment by Foxxite** [comment]
@Veltabex You could compile the source yourself, it's not that difficult.

**Comment by Veltabex** [comment]
Thanks works well had a little help from my friend to get it done, had a question, is the skin suppose to look like this ?
![image](https://user-images.githubusercontent.com/62775096/134804510-5d63dbed-0738-42ec-8c4f-b2f387ccf900.png)

& Is it possible to make each slot he related to new different world ?

**Comment by Veltabex** [comment]
Ignore my first question.

**Comment by Foxxite** [comment]
> Thanks works well had a little help from my friend to get it done, had a question, is the skin suppose to look like this ?

Yes, that's the default skin. Not sure if I made that a config option or not.

> & Is it possible to make each slot he related to new different world ?

Not supported out of the box, but you could code that in yourself.
